rotkehlchen/db/upgrades/v38_v39.py

```diff
@@ -261,6 +262,20 @@ def _add_arbitrum_one_location_and_nodes(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_arbitrum_one_location_and_nodes')
 
 
+def _remove_saddle_excahnge(write_cursor: 'DBCursor') -> None:
```

Comment:
excahnge -> exchange 

... but better yet. Oracle

```suggestion
def _remove_saddle_oracle(write_cursor: 'DBCursor') -> None:
```

Here and below